packages/now-cli/src/commands/env/rm.ts

```diff
@@ -0,0 +1,152 @@
+import cmd from '../../util/output/cmd';
+import withSpinner from '../../util/with-spinner';
+import { emoji, prependEmoji } from '../../util/emoji';
+
+type Options = {
+  '--debug': boolean;
+};
+
+export default async function rm(
+  ctx: NowContext,
+  opts: Options,
+  args: string[],
+  output: Output
+) {
+  const {
+    authConfig: { token },
+    config,
+  } = ctx;
+  const { currentTeam } = config;
+  const { apiUrl } = ctx;
+  const debug = opts['--debug'];
+  const client = new Client({ apiUrl, token, currentTeam, debug });
+  const link = await getLinkedProject(output, client);
+
+  if (link.status === 'error') {
+    return link.exitCode;
+  } else if (link.status === 'not_linked') {
+    output.print(
+      `${chalk.red(
+        'Error!'
+      )} Your codebase isn’t linked to a project on ZEIT Now. Run ${cmd(
+        'now'
+      )} to link it.\n`
+    );
+    return 1;
+  } else {
+    const { project } = link;
+    if (args.length > 2) {
+      output.error(
+        `Invalid number of arguments. Usage: ${chalk.cyan(
+          '`now env rm <key> <environment>`'
+        )}`
+      );
+      return 1;
+    }
+
+    let [envName, envTarget] = args;
+    let envTargets: ProjectEnvTarget[] = [];
+
+    if (envTarget) {
+      if (!isValidEnvTarget(envTarget)) {
+        output.error(
+          `The environment ${cmd(envTarget)} is not valid.\n` +
+            `Please use one of the following: <${validEnvTargets().join(
+              ' | '
+            )}>.`
+        );
+        return 1;
+      }
+      envTargets.push(envTarget);
+    }
+
+    while (!envName) {
+      const { inputName } = await inquirer.prompt({
+        type: 'input',
+        name: 'inputName',
+        message: `What’s the name of the variable?`,
+      });
+
+      if (!inputName) {
+        output.error(`Name cannot be empty`);
+        continue;
+      }
+
+      envName = inputName;
+    }
+
+    const envs = await getEnvVariables(output, client, project.id);
+    const existing = new Set(
+      envs.filter(r => r.key === envName).map(r => r.target)
+    );
+
+    if (existing.size === 0) {
+      output.error(`The environment variable ${cmd(envName)} was not found.\n`);
+      return 1;
+    }
+
+    if (envTargets.length === 0) {
+      const choices = getEnvTargetChoices().filter(c => existing.has(c.value));
+      if (choices.length === 0) {
+        output.error(
+          `The environment variable ${cmd(
+            envName
+          )} was found but it is not assign to any environments.\n`
+        );
+        return 1;
+      } else if (choices.length === 1) {
+        envTargets = [choices[0].value];
+      } else {
+        const { inputTargets } = await inquirer.prompt({
+          name: 'inputTargets',
+          type: 'checkbox',
+          message: `Remove ${envName} from which environments (select multiple)?`,
+          choices,
+        });
+        envTargets = inputTargets;
+      }
+    }
+
+    const yes = await promptBool(
```

Comment:
Can we add `-y` to bypass this prompt?

Reply:
I added the `--yes` flag to match domains